docs/user_guide.rst

```diff
@@ -94,90 +117,52 @@ quantum computer, and use Qiskit's :meth:`transpile` function followed by :meth:
     transpiled_circuit = transpile(qc, backend=backend)
     job = backend.run(transpiled_circuit, shots=1000)
 
-    print(job.result().get_counts())
 
 .. note::
 
-   If you want to inspect the circuits that are sent to the device, refer to `circuit_callback` option in the execution options table below. See also
-   `Inspecting the final circuits before submitting them for execution`_ for inspecting the actual run request sent for
-   execution. As of ``qiskit >= 1.0``, Qiskit no longer supports :meth:`execute`, but in all supported versions it is possible
-   to first transpile the ciruit and then run as shown in the code above. Alternatively, the method
-   :func:`transpile_to_IQM` can also be used to transpile circuits. In particular, when running circuits on devices with a
-   resonator, it is recommended to use :func:`transpile_to_IQM` instead of :meth:`transpile`.
-
-You can optionally set IQM backend specific options as additional keyword arguments to the :meth:`.IQMBackend.run` method. 
-For example, if you know an ID of a specific calibration set that
-you want to use, you can provide it as follows:
-
-.. code-block:: python
-
-    job = backend.run(qc, shots=1000, calibration_set_id="f7d9642e-b0ca-4f2d-af2a-30195bd7a76d")
-
-
-Alternatively, you can update the values of the options directly on the backend instance using the :meth:`.IQMBackend.set_options`
-and then call execution methods without specifying additional keyword arguments. You can view all available options and
-their current values using `backend.options`. Below table summarizes currently available options:
+   As of ``qiskit >= 1.0``, Qiskit no longer supports :meth:`execute`, but in all supported versions it is possible
+   to first transpile the circuit and then run as shown in the code above. Alternatively, the function
+   :func:`.transpile_to_IQM` can also be used to transpile circuits. In particular, when running circuits on devices with a
+   resonator, it is recommended to use :func:`.transpile_to_IQM` instead of :func:`transpile`.
```

Comment:
```suggestion
   computational resonator (IQM Star architecture), it is recommended to use :func:`.transpile_to_IQM` instead of
   :func:`transpile`.
```

Reply:
Fixed.